EIPS/eip-X-identity.md

```diff
@@ -0,0 +1,302 @@
+---
+eip: <to be assigned>
+title: ERC-725 Identity
```

Comment:
No need to include the EIP/ERC number in the title.

Reply:
I would also argue that the title is a bit broad.

Reply:
I renamed it to Proxy Identity to be more specific